ns1/data_source_zone.go

```diff
@@ -97,32 +92,3 @@ func dataSourceZone() *schema.Resource {
 		Read: zoneRead,
 	}
 }
-
-func dataSourceZoneToResourceData(d *schema.ResourceData, z *dns.Zone) error {
```

Comment:
Can you elaborate on why you deleted this function?

Reply:
This function is unused, on line 92 where it says `Read: zoneRead` - it references function `zoneRead` in the `resource_zone.go` file, that's used by this data source instead.